### PR TITLE
Add support for iotedged+Windows to quickstart

### PR DESCRIPTION
**Pull request by damonbarry**
- Adds support for Windows in the `-b iotedged` case
- Fixes a bug: when you don't provide your own deployment.json file, quickstart is building the deployment JSON incorrectly, causing a crash.

### CODE REVIEW

File: smoke/IotEdgeQuickstart/details/IotedgedWindows.cs [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+            this.archivePath = archivePath;
+            this.credentials = credentials;
+        }
+
+        public async Task VerifyNotActive()
+        {
+            bool active = true;
+            try
+            {
+                await Process.RunAsync("sc", "query iotedge");
```
Comment (avranju) [inline smoke/IotEdgeQuickstart/details/IotedgedWindows.cs]: Is this going to run on the host on Windows? If yes, we may be able to use the [service controller](https://docs.microsoft.com/en-us/dotnet/api/system.serviceprocess.servicecontroller?view=netframework-4.7.2) API for this. Its a .NET Framework only thing though.

Comment (damonbarry) [reply smoke/IotEdgeQuickstart/details/IotedgedWindows.cs]: Well yes this would run on the windows host, but we're compiling for .NET Core, so...

Comment (damonbarry) [reply smoke/IotEdgeQuickstart/details/IotedgedWindows.cs]: ...although the doc you linked to said it's supported on .NET Core as well.

Comment (damonbarry) [reply smoke/IotEdgeQuickstart/details/IotedgedWindows.cs]: This bit of code has been hassle-free so far so I'll leave it. But we can change it whenever it suites us.

Comment (avranju) [reply smoke/IotEdgeQuickstart/details/IotedgedWindows.cs]: Sounds good

**Pull request merged**